tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -732,6 +732,18 @@ Status ConvertAfterShapes(const ConversionParams& params) {
     segment_options.exclude_node_list.insert(node);
   }
   segment_options.minimum_segment_size = params.minimum_segment_size;
+
+  // This mechanism prevent the segmenter to provide the TRT Engine when doing
+  // INT8 calibration. This mechanism would be dangerous because the calibration
+  // phase is conducted in FP32 and may lead to different engines between the
+  // calibration phase in FP32 and the final INT8 engine.
```

Comment:
Let's change the comment to this:
We allow the segmenter to include or exclude a node based on the precision of the TRT engine we will build. We only know the precision of the TRT engine when calibration is not used. When calibration is used, we may build a FP32 engine or a INT8 engine for the same segment.

Reply:
@bixia1 done. Please run the CI ;)